installing/installing-fips.adoc

```diff
@@ -14,7 +14,9 @@ For more information about the NIST validation program, see link:https://csrc.ni
 
 [IMPORTANT]
 ====
-To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode, because {op-system-base} 9 is not currently supported.
```

Comment:
@dfitzmau I'm not if it would be better to remove the `because` word. If we're saying we must run the installation program from a RHEL computer, **because** RHEL 9 is not currently supported, this sounds like a bit confusing, RHEL contains RHEL 9.